.circleci/config.yml

```diff
@@ -45,8 +51,10 @@ jobs:
       - checkout
 
       - android/restore-gradle-cache
+          cache-prefix: << pipeline.parameters.gradle-cache-prefix >>
```

Comment:
Looks like you are getting validation errors in the CI build. I think you are missing a colon ":" after steps such as `android/restore-gradle-cache`, `android/save-gradle-cache`, etc.